docs/README.md

```diff
@@ -25,4 +25,15 @@ tbd
 tbd
 
 ## Uninstalling
-tbd
+
+To uninstall the `sail-operator` from the cluster, you can execute the following command. This will remove all Istio CRDs, the sail-operator deployment, and the associated namespace.
+
+```shell
+make undeploy
```

Comment:
As this is end user documentation, I think we should not expect installation to have happened from source, so we need to tell people to remove the subscription in OLM and take care of any residual resources (I don't think we have that problem in sail though- so it's probably just the OLM part)

Reply:
Sure @dgn. Let me try once with OCP just to be sure that we are not leaving any artifacts. Will update the PR once I have the details.

Reply:
@dgn, after uninstalling the Istio, IstioCNI resources, the sail-operator and the projects (istio-system, istio-cni) the cluster still has the following CRDs.

```
$ oc get crds -A | grep istio
authorizationpolicies.security.istio.io                           2024-06-18T08:27:28Z
destinationrules.networking.istio.io                              2024-06-18T08:27:29Z
envoyfilters.networking.istio.io                                  2024-06-18T08:27:28Z
gateways.networking.istio.io                                      2024-06-18T08:27:28Z
istiocnis.operator.istio.io                                       2024-06-18T08:27:30Z
istiorevisions.operator.istio.io                                  2024-06-18T08:27:28Z
istios.operator.istio.io                                          2024-06-18T08:27:30Z
peerauthentications.security.istio.io                             2024-06-18T08:27:31Z
proxyconfigs.networking.istio.io                                  2024-06-18T08:27:30Z
requestauthentications.security.istio.io                          2024-06-18T08:27:28Z
serviceentries.networking.istio.io                                2024-06-18T08:27:29Z
sidecars.networking.istio.io                                      2024-06-18T08:27:27Z
telemetries.telemetry.istio.io                                    2024-06-18T08:27:28Z
virtualservices.networking.istio.io                               2024-06-18T08:27:28Z
wasmplugins.extensions.istio.io                                   2024-06-18T08:27:30Z
workloadentries.networking.istio.io                               2024-06-18T08:27:30Z
workloadgroups.networking.istio.io                                2024-06-18T08:27:31Z
sgaddam@fedora:~/deployk8s/crc$ 
```

Reply:
Should we add a last step for this cleaning up?

```
kubectl get crd -oname | grep istio.io | xargs kubectl delete
```

Reply:
Sure, we can do that. Thanks.

Reply:
hmm it looks like OLM leaves it to the users whether to remove CRDs: https://olm.operatorframework.io/docs/tasks/uninstall-operator/#step-4-deciding-whether-or-not-to-delete-the-crds-and-apiservices

We can reference that and provide a command that removes the CRDs if wanted

Reply:
Thanks @dgn and @bmangoen for reviewing. Let me add a reference to the OLM note and the manual command to delete the CRDs.